test/functional/applications_controller_test.rb

```diff
@@ -211,6 +263,47 @@ class ApplicationsControllerTest < ActionController::TestCase
         end
       end
     end
+
+    context "when there is a kubernetes API response error" do
+      setup do
+        error = lambda { |environment: "test", repo_name: "Application", status: "Running"| # rubocop:disable Lint/UnusedBlockArgument
```

Comment:
I can’t see these arguments used anywhere, can we simply remove them from the lambda definition?

```suggestion
        error = -> { raise Kubeclient::HttpError.new(408, "Timeout message", "Timeout response") }
```

We should not be silencing cops. 
Btw you can run `govuk-docker run release-lite bundle exec rubocop -A` to auto correct some linting errors.